activestorage/test/dummy/config/initializers/cookies.rb

```diff
@@ -5,3 +5,7 @@
 # Specify a serializer for the signed and encrypted cookie jars.
 # Valid options are :json, :marshal, and :hybrid.
 Rails.application.config.action_dispatch.cookies_serializer = :json
+
+# Specify the SameSite level protection for the cookies
+# Valid options are :no_protection, :lax, and :strict.
+Rails.application.config.action_dispatch.cookies_same_site_protection = :no_protection
```

Comment:
This config is feeling like a special case for one type of default cookie option.

How would it compare with offering config for cookie defaults, and setting `same_site: nil` as the default?